tests/samplers/test_rejection_sampler.py

```diff
@@ -339,7 +339,7 @@ def __init__(self, vocab_size: int, rejection_sampler: RejectionSampler):
         self.vocab_size = vocab_size
         self.vocab_range = (0, vocab_size)
 
-        self.rejection_sampler.init_gpu_tensors(rank=0)
+        self.rejection_sampler.init_gpu_tensors(device=0)
```

Comment:
Should it be `device=device` here?

Reply:
> Should it be `device=device` here?

there is no `device` in this class, so I assume all things happen on gpu 0, correct me if I'm wrong.